04-advanced-topics/validate_differential_expression_adv_topics_00_author_de.Rmd

```diff
@@ -180,10 +180,10 @@ test.probe <- author.df %>%
 # Make a dataframe with the group information
```

Comment:
Same comment re explicitly attaching the `ggplot2` library at the top of this notebook.

Reply:
Same.